udata/core/dataset/models.py

```diff
@@ -148,6 +149,48 @@ def check_availability(self, group):
     def is_available(self):
         return self.check_availability(group=None)
 
+    @cached_property
+    def json_ld(self):
+
+        result = {
+            '@type': 'DataDownload',
+            '@id': str(self.id),
+            'url': self.url,
+            'name': self.title or _('Nameless resource'),
+            'contentUrl': self.url,
+            'dateCreated': self.created_at.isoformat(),
+            'dateModified': self.modified.isoformat(),
+            'datePublished': self.published.isoformat(),
+        }
+
+        if 'views' in self.metrics.keys():
```

Comment:
I agree with @davidbgk : `'string' in dict` is used everywhere accross the project.
Given the fact that it's `self.metrics` it means you inside the class so you know which structure is `metrics`.
In udata, `self.metrics` is always a dict.